### DISCUSSION TITLE
Welcome to sprinto Discussions!

### DISCUSSION THREAD

**Discussion by pengowray** [pengowray/sprinto#6] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by TheLoveliestTurtle** [comment]
Hello! I'm trying to use Sprinto bot. All the rest of the members in the server can use the bot just fine but I can only initiate sprints, I can never join them. I will type in /join and the bot will simply not respond. Yet the bot responds just fine when I type /sprint. Is there any way to fix this?

**Comment by pengowray** [comment]
Hello and happy new year, 

Unfortunately a lot of people have trouble with the how Discord's slash command UI works and there's no simple fix for it. (And they don't let bots use the old prefix commands anymore, so I can't go back to that)

There's no reason that /sprint should work but not /join. Try pausing for a second after typing /join so it has time to register. You can also try `@sprinto join` Some people find that easier or more reliable.

If you want to dive in deeper, I made this post trying to cover the various issues that come up: https://www.patreon.com/posts/71439355

Hope that helps.

**Reply by TheLoveliestTurtle** [reply]
Alright, I’ll try that out. Happy New Year and thanks for the help!
